src/components/Combobox/components/Listbox.tsx

```diff
@@ -0,0 +1,64 @@
+import { ReactElement, useEffect } from "react";
+import styled from "styled-components";
+
+import { IListboxProps } from "../types";
+
+const StyledListbox = styled.div<{ $isExpanded?: boolean }>`
+  position: absolute;
+  top: 2rem;
+  left: 0;
+  display: ${(props) => (props.$isExpanded ? "flex" : "none")};
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  max-height: 35vh;
+  border: 1px solid ${(props) => props.theme.colors.grey300};
+  border-radius: 3px;
+  background-color: ${(props) => props.theme.colors.babyPowder};
+  overflow-y: auto;
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 2px 4px;
+  z-index: 2;
+`;
+
+const Listbox = <T extends unknown>(
+  props: IListboxProps<T>
+): ReactElement | null => {
```

Comment:
Using this form of typings instead as `React.FC<Props>` does not support TS generics.